content/_index.md

```diff
@@ -49,31 +39,43 @@ By correlating every piece of software to a verifiable, complete, and concise ar
 * Post exploit forensics -->
 {{< /home/section >}}
 
-{{< home/section title="How?">}}
-Drawing on the version control system `git`, GitBOM observes that:
+<!--
+{{< home/section title="How does it work?">}}
+GitBOM draws much inspiration from the version control system `git` and observes that:
 
-1. Every artifact is a [blob](/resources/glossay/#merkle-tree)
-2. Every blob has a [gitref](/resources/glossay/#merkle-tree)
-3. Therefore the gitref may be used as an artifact id for an artifact
-4. Most source code artifacts are stored already with their gitref as their id
+1. Every file, regardless of type, can be stored as a binary object.
+2. Hash functions, such as SHA1 and SHA256, are sufficiently unique that their output may be used to uniquely identify any binary input.
+3. Git references all files in a git tree by a *gitref*, which is the output of hashing the binary representation of that file.
+4. Git stores each commit in a file containing an ordered list of *gitrefs* of files in that commit. This is known as the *commit ref*.
+5. Git represents a branch as an ordered list of *gitrefs* of the *commit refs* of commits in that branch.
```

Comment:
I feel like bringing branches in here, on the home page, is just overloading with concepts that aren't needed to talk about GitBOMs.

Reply:
Yeah, agreed. This whole section is currently commented out because, after I wrote it, I had the same thought. It's too much info for the front page.